opm/output/Cells.hpp

```diff
@@ -32,6 +32,10 @@ class SimulationDataContainer;
 namespace data {
 
     struct CellData {
+        CellData(std::string name, UnitSystem::measure dim, std::vector<double> data)
```

Comment:
> Interesting, I wonder why?

My guess would be he relied on a vector or other container of these and tried to use `emplace_back`. Using `struct foo` from my example above:

```
std::string x; std::vector< int > y;
vec.emplace_back( x, y ); // won't compile
vec.emplace_back( { x, y } ); // won't compile
vec.emplace_back( foo{ x, y } ); // compiles
```